vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -0,0 +1,62 @@
+package life.qbic.datamanager.views.project.view.components;
+
+import com.vaadin.flow.component.Composite;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.formlayout.FormLayout.ResponsiveStep;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import java.io.Serial;
+import java.util.Objects;
+import life.qbic.datamanager.views.components.CardLayout;
+import life.qbic.projectmanagement.application.ProjectInformationService;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public class ProjectDetailsComponent extends Composite<CardLayout> {
```

Comment:
For this card a title is missing. Something like "Project Information" or so to be displayed as Card-Title. I think there is a specific method .addTitle() or so for this

Reply:
Thanks, I addressed it :)